base/error.go

```diff
@@ -74,6 +74,12 @@ var (
 
 	// ErrReplicationLimitExceeded is returned when then replication connection threshold is exceeded
 	ErrReplicationLimitExceeded = &sgError{"Replication limit exceeded. Try again later."}
+
+	// ErrSkippedSequencesMissing is returned when attempting to remove a sequence range form the skipped sequence list and at least one sequence in that range is not present
+	ErrSkippedSequencesMissing = &sgError{"Sequence range has sequences that aren't present in skipped list"}
```

Comment:
Think this error has been mistakenly been pulled in, don't think its needed for 3.2.4